.github/workflows/ci.yml

```diff
@@ -1244,22 +1239,17 @@ jobs:
     runs-on: windows-latest
 
     steps:
-      - uses: timfel/setup-python@fc9bcb4a04f5b1ea7d678c2ca7ea1c479a2468d7
-        with:
-          python-version: "graalpy24.1"
-
       - name: "Download binary"
         uses: actions/download-artifact@95815c38cf2ff2164869cbab79da8d1f422bc89e # v4.2.1
         with:
           name: uv-windows-x86_64-${{ github.sha }}
 
-      - name: Graalpy info
-        run: Get-Command graalpy
+      - name: "Install GraalPy"
+        run: .\uv.exe python install graalpy
 
       - name: "Create a virtual environment"
         run: |
-          $graalpy = (Get-Command graalpy).source
-          .\uv.exe venv -p $graalpy
+          .\uv.exe venv -p graalpy --python-preference only-managed
```

Comment:
This should be `--managed-python` instead of `--python-preference only-managed` now.

Reply:
Changed